### PR TITLE
[PBA-4618] FullScreen event handling

### PR DESCRIPTION
**Pull request by akshaydch11**

### CODE REVIEW

File: sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m [hunk 1]
```diff
@@ -344,4 +347,10 @@ - (void)playPauseFromAdTappedNotification {
 - (void)onApplicationDidBecomeActive:(NSNotification *)notification {
   MACaptionAppearanceSetDisplayType(kMACaptionAppearanceDomainUser, kMACaptionAppearanceDisplayTypeForcedOnly);
 }
+
+- (void)notifyFullScreenChange:(BOOL) isFullScreen {
+    [[NSNotificationCenter defaultCenter] postNotificationName:OOOoyalaPlayerFullScreenChangedNotification object:self
+        userInfo:[[NSDictionary alloc] initWithObjectsAndKeys:@(isFullScreen), @"fullScreen", nil]];
```
Comment (brol1dev) [inline sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m]: So, I know you're new to Obj-C but could you change the NSDictionary initialization? There's a shorter syntax, let me show you:

```
// Long old initialization
[[NSDictionary alloc] initWithObjectsAndKeys:@(isFullScreen), @"fullScreen", nil];
// new good looking initialization :)
@{@"fullscreen": @(isFullscreen)};
```

**Review by brol1dev — APPROVED** [review]
All good, I just added a comment to change the NSDictionary initialization.

File: sdk/iOS/OoyalaSkinSDK/OOSkinViewController.h [hunk 2]
```diff
@@ -19,6 +19,10 @@
 @property (nonatomic, readonly) NSString *version;
 @property (nonatomic, readonly) OOClosedCaptionsStyle *closedCaptionsDeviceStyle;
 
+// notifications
+extern NSString *const OOOoyalaPlayerFullScreenChangedNotification; /**< Fires when player goes FullScreen  */
```
Comment (wunnath) [inline sdk/iOS/OoyalaSkinSDK/OOSkinViewController.h]: I think all our notifications are declared in OOOoyalaPlayer.h in OoyalaSDK.

File: sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m [hunk 3]
```diff
@@ -52,6 +52,8 @@ @implementation OOSkinViewController
 static NSString *kFrameChangeContext = @"frameChanged";
 static NSString *kViewChangeKey = @"frame";
 
+NSString *const OOOoyalaPlayerFullScreenChangedNotification = @"fullScreenChanged";
```
Comment (wunnath) [inline sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m]: this should be moved to OOOoyalaPlayer.m in OoyalaSDK.

Comment (lental) [reply sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m]: sorry @wunnath  i disagree, this notification is specifically about the skin.

I would actually suggest a rename of "OOSkinViewControllerFullscreenChangedNotification"

Comment (akshaydch11) [reply sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m]: @wunnath Yes these are specific to SkinSDK and so they are here :)
@lental I will change name :)

Comment (wunnath) [reply sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m]: got it. thanks @lental  and @akshaydch11

**Pull request merged**